boxsdk/util/chunked_uploader.py

```diff
@@ -46,6 +47,21 @@ def start(self):
         content_sha1 = self._sha1.digest()
         return self._upload_session.commit(content_sha1=content_sha1, parts=self._part_array)
 
+    def abort(self):
+        """
+        Abort an upload session, cancelling the upload and removing any parts that have already been uploaded.
+
+        :returns:
+            A boolean indication success of the upload abort.
+        :rtype:
+            `bool`
+        """
+        self._content_stream = None
+        self._part_array = []
+        self._inflight_part = None
+        self._is_aborted = True
+        return self._upload_session.abort()
```

Comment:
I think this is the only line you need for this method. The rest are handled by GC